src/CMakeLists.txt

```diff
@@ -49,3 +49,7 @@ if(CMAKE_TESTING_ENABLED)
     add_subdirectory(test)
 endif(CMAKE_TESTING_ENABLED)
 
+if (NOT HAVE_MICROSOFT_GMTIME_S AND NOT HAVE_GMTIME_S AND NOT HAVE_GMTIME_R)
```

Comment:
I think this check should be located in either ConfigureChecks.cmake or /src/compat/CMakeLists.txt

Also, there should be another check for the `localtime_*()` functions.